src/M6Web/Bundle/LogBridgeBundle/Config/Configuration.php

```diff
@@ -9,14 +9,12 @@
  */
 class Configuration
 {
-    private ?FilterCollection $filters;
+    private ?FilterCollection $filters = null;
 
-    private ?array $activeFilters;
+    private ?array $activeFilters = null;
 
     public function __construct()
```

Comment:
I think you can remove the constructor if nothing is done here

Reply:
Absolutely !